packages/teams-js/src/public/pages.ts

```diff
@@ -690,4 +690,55 @@ export namespace pages {
       return runtime.supports.pages ? (runtime.supports.pages.appButton ? true : false) : false;
     }
   }
+
+  /**
+   * Parameters for the NavigateToSelf API
+   */
+  export interface NavigateToSelfParams {
+    /**
+     * Developer-defined ID of the Page to navigate to within the application (Formerly EntityID)
+     */
+    pageId: string;
+
+    /**
+     * Optional developer-defined ID describing the content to navigate to within the Page. This will be passed
+     * back to the application via the Context object.
+     */
+    subPageId?: string;
+  }
+
+  export namespace self {
```

Comment:
We should add a comment describing what kind of functionality this namespace contains (functions for navigating without your own application) and mark it as @beta